extensions/cornerstone/src/panels/PanelMeasurement.tsx

```diff
@@ -71,27 +87,31 @@ export default function PanelMeasurementTable({
         ref={measurementsPanelRef}
         data-cy={'trackedMeasurements-panel'}
       >
-        <MeasurementTable
-          key="tracked"
-          title="Measurements"
-          data={measurements}
-          {...onArgs}
-          // onColor={changeColorMeasurement}
-        >
-          <MeasurementTable.Header>
-            {customHeader && (
-              <>
-                {typeof customHeader === 'function'
-                  ? customHeader({
-                      additionalFindings,
-                      measurements,
-                    })
-                  : customHeader}
-              </>
-            )}
-          </MeasurementTable.Header>
-          <MeasurementTable.Body />
-        </MeasurementTable>
+        {Array.from(measurements).map(([key, value]) => {
+          return (
+            <MeasurementTable
+              key={`tracked-${key}`}
```

Comment:
entThis isn't necessarily tracked on this version of it, so you shouldn't be referencing tracked here, but should just get something passed in from the parent about what you are naming this.